timescaledb/tutorials/nfl-analytics/advanced-analysis.md

```diff
@@ -15,7 +15,7 @@ Python module, which is one of many great visualization tools.
 
 ### Average yards run for a player over a game
 
-This query uses a percentile approximation [hyperfunction](/hyperfunctions/percentile-approximation/) 
+This query uses a percentile approximation [hyperfunction](https://docs.timescale.com/api/latest/hyperfunctions/percentile-approximation/percentile-approximation) 
```

Comment:
@Loquacity I just put in the hard coded link because the referencing one didn't work with the api folder being before the latest. If you have a better solution please feel free to update these! Thanks in advance for looking this over!

Reply:
hrm, you should be able to use `/api/latest/percentile ...`

Reply:
Ahh, okay, I just wasn’t redirecting it right then haha. I’ll update the links to follow that pattern! Thank you Lana!!